cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,21 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+PROJECT?="google_containers"
+REGISTRY?="gcr.io"
+
 all:
-	docker build --pull -t gcr.io/$(PROJECT)/kubemark .
+	gcloudpush
+
+build:
+	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+
+gcloudpush:
+	build
 	gcloud docker -- push gcr.io/$(PROJECT)/kubemark
```

Comment:
Since we do default to gcr.io anyway, why can't the registry be a variable as well. Besides, this could change in the future as well (maybe).